src/plugins/kibana_legacy/public/styles/_paginate.scss

```diff
@@ -0,0 +1,57 @@
+aginate {
```

Comment:
Missing the `p`

Reply:
I'm also pretty sure this is only used by the `agg_table`, can we move this there?

Reply:
thanks for the `p` :) it's also used in timelion: https://github.com/kertal/kibana/blob/kertal-2020-06-17-cleanup-legacy-styles/src/legacy/core_plugins/timelion/public/directives/saved_object_finder.html#L56

Reply:
moved it to `src/plugins/kibana_legacy/public/paginate`